src/main.js

```diff
@@ -2,7 +2,9 @@
 
 const { createServer } = require('./createServer');
 
+const PORT = 3000;
+
 createServer().listen(3000, () => {
```

Comment:
Consider using the `PORT` constant instead of hardcoding `3000` in the `listen` method to maintain consistency and make it easier to change the port number in the future.